Sprint-3/revise/investigate/find.js

```diff
@@ -20,6 +20,16 @@ console.log(find("code your future", "z"));
 // Pay particular attention to the following:
 
 // a) How the index variable updates during the call to find
+    //  The index will increase with each iteration of the while look, 
+    //  unless the condition/s in the if statement are met.
+    // If the conditions are met, there is a return. The return would be the value of index.
+
 // b) What is the if statement used to check
+    //  The if statement check if the character at current index of the string, is the same as the input value/variable char.
+
 // c) Why is index++ being used?
+    // The ++ operator is the increment.  It mean to "add 1 to", 
+    // so what index++ does is add 1 to index with each iteration of the while loop.
```

Comment:
This explains what `index++` do, but it does not give any insight why `index++` is used (as oppose to `index+=2`, for example).

Reply:
I've added additional information.
`The reason for this so that each character of the string can be checked.`